docusaurus/docs/reactnative/common-content/core-components/channel-list/props/empty_state_indicator.mdx

```diff
@@ -1,5 +1,5 @@
-Custom UI component for replacing default empty state indicator.
+Rendered when the channel list is empty and not loading via the [ListEmptyComponent](https://reactnative.dev/docs/flatlist#listemptycomponent) prop on the FlatList.
 
 | Type | Default |
 | - | - |
-| Component, element | [EmptyStateIndicator](https://github.com/GetStream/stream-chat-react-native/blob/master/package/src/components/Indicators/EmptyStateIndicator.tsx) |
+| component | [EmptyStateIndicator](https://github.com/GetStream/stream-chat-react-native/blob/master/package/src/components/Indicators/EmptyStateIndicator.tsx) |
```

Comment:
@vishalnarkhede @nhannah should we be using `Component` or `component?`

Reply:
lowercase